MNIST Dataset Classification/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy
```

Comment:
Add the versions of the libraries

Reply:
kk .... i did all the changes can u check it